core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/parser/synthetic/JdkTypeIDsPreJdk11.java

```diff
@@ -39,7 +39,7 @@
  * Contains type IDs for events that are produced by JDK 7 and 8.
  */
 @SuppressWarnings({"nls", "unused"})
-final class JdkTypeIDsPreJdk11 {
+public final class JdkTypeIDsPreJdk11 {
```

Comment:
Made public to be able to use it from here: https://github.com/openjdk/jmc/blob/9f9665812a80339782c2e00be83b041954ee766b/core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/internal/parser/v0/EventParserManager.java#L100

Reply:
If we make that class public, we might as well rename it. Since the JDK 8 backport, it really is Oracle type IDs - so perhaps OracleJdkTypeIDsPre11?

Reply:
done